src/packages/core/data-type/workspace/data-type-workspace.context.ts

```diff
@@ -146,8 +145,46 @@ export class UmbDataTypeWorkspaceContext
 		return this._configDefaultData?.find((x) => x.alias === alias)?.value;
 	}
 
-	createVariantContext(host: UmbControllerHost): UmbDataTypeVariantContext {
-		return new UmbDataTypeVariantContext(host, this);
+	createPropertyDatasetContext(host: UmbControllerHost) {
```

Comment:
If I understand this correctly would it then be possible to make this as a "sub context" or use the property-dataset element to achieve the same?

Reply:
We had a good talk and I'm moving the creation of this context to the editor implementation. Not a sub context, but a more local implementations.